engineio/src/asynchronous/async_transports/polling.rs

```diff
@@ -80,14 +81,21 @@ impl AsyncTransport for PollingTransport {
         Ok(())
     }
 
-    async fn poll(&self) -> Result<Bytes> {
-        Ok(self
-            .client
-            .get(self.address().await?)
-            .send()
-            .await?
-            .bytes()
-            .await?)
+    async fn stream(&self) -> Result<Pin<Box<dyn Stream<Item = Result<Bytes>> + '_>>> {
```

Comment:
suggestion: return impl Stream

Reply:
This is apparently not possible, as `impl "Trait"` is not allowed in a trait definition at the moment...

Reply:
OH right, forgot about that.